src/guide/migration/emits-option.md

```diff
@@ -0,0 +1,62 @@
+---
+title: emits Option
+badges:
+  - new
+---
+
+# `emits` Option <MigrationBadges :badges="$frontmatter.badges" />
+
+## Overview
+
+Vue 3 now offers an `emits` option similar to the existing `props` option. This option can be used to define the events that a component can emit to its parent.
+
+## 2.x Behavior
+
+In Vue 2, you can define the props that a component received, but you can't define which ebents it can emit:
+
+```javascript
+<template>
+  <p>{{ text }}</p>
+  <button v-on:click="$emit('accepted')">OK</button>
+</template>
+<script>
+  export default {
+    props: ['text'],
+  }
+</script>
+```
+
+## 3.x Behavior
+
+Similar to props, the events that the component emit can now be defined with the `emits` option.
+
+```javascript
+<template>
+  <p>{{ text }}</p>
+  <button v-on:click="$emit('accepted')">OK</button>
+</template>
+<script>
+  export default {
+    props: ['text'],
+    emits: ['accepted']
+  }
+</script>
+```
+
+The option also excepts an object notation, which allows to define validators for the arguments that have passed with the emitted event, similar to validators in props definitions.
```

Comment:
*excepts* -> *accepts*

There seems to be something missing around *'allows to define'*. Perhaps *'allows you to define'*?

Also, perhaps *'are passed'* rather than *'have passed'*? Or maybe *'have been passed'*?